lib/datasource/maven/index.ts

```diff
@@ -130,8 +118,8 @@ async function filterMissingArtifacts(
       .filter(([_, artifactUrl]) => Boolean(artifactUrl))
       .map(
         ([version, artifactUrl]) =>
-          (): Promise<ArtifactInfoResult> =>
-            getArtifactInfo(version, artifactUrl)
+          async (): Promise<ArtifactInfoResult> =>
```

Comment:
Do we need the return type here?
```suggestion
          async () =>
```